conda-store-server/conda_store_server/_internal/schema.py

```diff
@@ -129,7 +129,7 @@ class Config:
 
     @classmethod
     def from_list(cls, lst):
-        return cls(**{k: v for k, v in zip(cls.__fields__.keys(), lst)})
+        return cls(**{k: v for k, v in zip(cls.__fields__.keys(), lst, strict=False)})
```

Comment:
At some point I touched this file but in later commits I removed it since I didn't change anything. Now `pre-commit.ci` insists on linting this even though it's identical to what is on `main` :/

Reply:
seems ok now?